UEBPrep/app/models/card.rb

```diff
@@ -0,0 +1,28 @@
+class Card < ActiveRecord::Base
+  validates :content_type_id, presence: true
+  validates :content_id, presence: true
+  validates :created_by, presence: true
+  validates_uniqueness_of :content_id, :scope => :content_type_id
```

Comment:
Do the content id's really need to be unique, or is it okay for multiple cards to have some common content?